faros_event.sh

```diff
@@ -0,0 +1,470 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Requires curl
+which curl &> /dev/null || { echo "curl required" & exit 1; }
```

Comment:
Isn't this `curl is required`? Same below

Reply:
I think it would be nice to print the whole list, e.g.
```
Error: curl is required
Please make sure curl, jq, etc, etc are available before running the script.   
```
(you can probably formulate this better)